### PR TITLE
Fix syntax error in scheme example

### PR DESCRIPTION
**Pull request by ChristopherHogan**
When I run `examples/holey-wvg-cavity.ctl` I get this error:
```
chris@ubuntu:$ ./meep ../../examples/holey-wvg-cavity.ctl 
ice-9/psyntax.scm:1274:12: In procedure dobody:
ice-9/psyntax.scm:1274:12: Syntax error:
../../examples/holey-wvg-cavity.ctl:73:6: definition in expression context,
where definitions are not, in form (define trans (add-flux fcen df nfreq 
(make flux-region (center (- (* 0.5 sx) dpml 0.5) 0) (size 0 (* w 2)))))
```
This PR fixes that.

Guile version:
```
chris@ubuntu: $ guile -v
guile (GNU Guile) 2.0.11
Packaged by Debian (2.0.11-deb+1-10)
Copyright (C) 2014 Free Software Foundation, Inc.
```
@stevengj @oskooi @HomerReid

### CODE REVIEW

**Comment by stevengj** [comment]
It would be more Scheme-like to define a local variable with `let`.  Don't `define trans` at all, and in place of `set! trans` do:
```scm
(let ((trans (add-flux ....)))
   (run-sources+ ...)
   (display-fluxes trans) ; print out the flux spectrum
)
```

**Comment by ChristopherHogan** [comment]
Fixed.

File: examples/holey-wvg-cavity.ctl [hunk 1]
```diff
@@ -70,10 +70,10 @@
...
       (set! symmetries (list (make mirror-sym (direction Y) (phase -1))))
       
-      (define trans ; transmitted flux
+      (let ((trans ; transmitted flux
 	(add-flux fcen df nfreq
 		  (make flux-region
-		    (center (- (* 0.5 sx) dpml 0.5) 0) (size 0 (* w 2)))))
+		    (center (- (* 0.5 sx) dpml 0.5) 0) (size 0 (* w 2))))))
       
       (run-sources+ (stop-when-fields-decayed
```
Comment (stevengj) [inline examples/holey-wvg-cavity.ctl]: This should be indented since it is inside the `let`.

**Comment by ChristopherHogan** [comment]
Sorry.  The combination of tabs and spaces, along with my editor's default tab settings mangled the indentation.  Fixed.

**Pull request merged**